keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/TestController.kt

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.yaml.APPLICATION_YAML_VALUE
+import com.netflix.spinnaker.kork.exceptions.SystemException
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/test"])
+class TestController {
```

Comment:
Consider renaming this to `ErrorSimulationController` for clarity.

Reply:
Will do.

Reply:
Addressed in 595a47f